libs/cart/state/src/models/stateful-cart-item.ts

```diff
@@ -0,0 +1,18 @@
+import { DaffCartItem } from '@daffodil/cart';
+
+/**
+ * The state of the cart item is intended to enhance the client side UX like indicating when a cart
+ * item was recently added/updated. For states that indicate the completion of some process, the state is given 
+ * a decay time based on the DaffCartItemStateDebounceTime injection token. For example when a cart item is
+ * added to the cart, the state of that item will be "New" for a designated time then will revert to the default state.
+ */
+export interface DaffStatefulCartItem extends DaffCartItem {
```

Comment:
In my mind, `DaffCartItemEntity` just says that the model is held in ngrx entity state. But anything that can be stored as an entity should be stored as an entity, so I don't think `DaffCartItemEntity` really tells you any extra information, because cart items should be expected to be an entity. On the other hand, `DaffStatefulCartItem` says that it is a cart item that has a state. Is there another reason you prefer `DaffCartItemEntity`?

Reply:
No actually I was misunderstanding what you meant by "stateful". I agree that your name is more informative and therefore better. However, the terminology gets confusing with the entity's state and ngrx state. What do you think about "status" instead of "state"?

Reply:
oh, I see where the misunderstanding was. 

"status" is fine with me. I'll make that change

Reply:
changed

Reply:
I just realized that we have a `DaffStatusEnum` in the design package. We should think about if this conflict is a problem. Should we change the name of `DaffStatefulCartItem`? How about `DaffStatusableCartItem`?

Reply:
reverted this back to stateful and daffState